app/managers.py

```diff
@@ -0,0 +1,49 @@
+from models import Actor
+import sqlite3
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self.connect_ = sqlite3.connect("cinema.sqlite")
+        self.tabel_name = "actors"
```

Comment:
There is a typo in the variable name `tabel_name`. It should be `table_name` to correctly represent its purpose.